aten/src/ATen/native/cuda/Nonzero.cu

```diff
@@ -183,6 +214,84 @@ void nonzero_cuda_out_impl(const Tensor& self, Tensor& out) {
   }
 }
 
+template <typename scalar_t>
+void nonzero_static_cuda_out_impl(
+    const Tensor& self,
+    int64_t size,
+    int64_t fill_value,
+    Tensor& out) {
+# if (defined(CUDA_VERSION) && CUDA_VERSION > 11040) || defined(USE_ROCM)
+
+  Tensor self_ = self.contiguous();
```

Comment:
Nit: consider naming the input argument self_ and the contiguous self so that you consistently use self (I audited the sites and you did it all correctly; but still, nice to make it harder to make the mistake lol).